conda-forge.yml

```
@@ -3,11 +3,11 @@ build_platform:
   linux_ppc64le: linux_64
```

```suggestion
  linux_ppc64le: linux_ppc64le
```

We can also try this (and rerender) if cross compilation fails.